tests/installation/src/utils.ts

```diff
@@ -20,7 +20,7 @@ import {
   ZOWE_FMID,
   ANSIBLE_ROOT_DIR,
   SANITY_TEST_REPORTS_DIR,
-  INSTALL_TEST_REPORTS_DIR,
+  INSTALL_TEST_REPORTS_DIR, SECURITY_SYSTEM_ACF2, SECURITY_SYSTEM_RACF, SECURITY_SYSTEM_TSS,
```

Comment:
Now the security system has been moved to the host vars I don't think these need to be imported

Reply:
we dont, I removed them